### PR TITLE
feat:detail page

### PR DESCRIPTION
**Pull request by florashore**

### CODE REVIEW

File: src/pages/proposalList/proposalList.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import ProposalShow from "../../components/proposalShow/proposalShow";
```
Comment (coderprovider) [inline src/pages/proposalList/proposalList.js:2]: component good

File: src/pages/proposalList/proposalList.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  const { contract } = useSelector((state) => state.contract)
+  const { account } = useSelector((state) => state.account)
+  const [lockAddress, setLockAddress] = useState('')
+  const [lockDuration, setLockDuration] = useState('')
+  const [maxLockAmount, setMaxLockAmount] = useState('')
+  const [LockedAmount, setLockedAmount] = useState('')
+
+  const getContractData = async () => {
+
+    const resLockAddress = await contract.methods.lockAddress().call()
```
Comment (coderprovider) [inline src/pages/proposalList/proposalList.js:20]: integrate?

File: src/pages/proposalList/proposalList.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            holders="150"
+            tokenVolume={(maxLockAmount.toString() * 2469.39).toFixed(3)}
+            networkAvatar="ethereum_img"
+            networkName="Ethereum"
+          />
+        </div>
+
+        <div className="mt-[20px] overflow-y-scroll w-[102%]">
+          <div className="flex flex-col gap-[10px]  h-[420px] pr-[13px]">
+            {
```
Comment (coderprovider) [inline src/pages/proposalList/proposalList.js:68]: 😀

**Review by coderprovider — COMMENTED** [review]
have to integrate

**Comment by florashore** [comment]
yeah

**Review by coderprovider — COMMENTED** [review]
others good

**Comment by florashore** [comment]
good

**Review by coderprovider — COMMENTED** [review]
table responsive ?

**Comment by miraclaers** [comment]
completed

**Comment by florashore** [comment]
😀

File: src/pages/proposalList/proposalList.js (hunk 1 above)
Comment (miraclaers) [reply src/pages/proposalList/proposalList.js:2]: 👍

File: src/pages/proposalList/proposalList.js (hunk 2 above)
Comment (miraclaers) [reply src/pages/proposalList/proposalList.js:20]: ♥

File: src/pages/proposalList/proposalList.js (hunk 3 above)
Comment (miraclaers) [reply src/pages/proposalList/proposalList.js:68]: 😀

**Review by miraclaers — COMMENTED** [review]
good

File: src/pages/proposalList/proposalList.js (hunk 1 above)
Comment (Jermaine522) [reply src/pages/proposalList/proposalList.js:2]: 😁

File: src/pages/proposalList/proposalList.js (hunk 2 above)
Comment (Jermaine522) [reply src/pages/proposalList/proposalList.js:20]: ♥

File: src/pages/proposalList/proposalList.js (hunk 3 above)
Comment (Jermaine522) [reply src/pages/proposalList/proposalList.js:68]: 😀

**Review by Jermaine522 — COMMENTED** [review]
😀

**Comment by florashore** [comment]
will merge

**Comment by florashore** [comment]
merged

**Pull request merged**